app/handlers/block_factory.py

```diff
@@ -0,0 +1,48 @@
+from app.handlers.blocks.content import Content
+from app.handlers.blocks.question import Question
+from app.handlers.blocks.list_collector import ListCollector
+from app.handlers.blocks.list_add_question import ListAddQuestion
+from app.handlers.blocks.list_edit_question import ListEditQuestion
+from app.handlers.blocks.list_remove_question import ListRemoveQuestion
+from app.handlers.blocks.primary_person_list_collector import PrimaryPersonListCollector
+from app.handlers.blocks.primary_person_question import PrimaryPersonQuestion
+from app.handlers.blocks.relationship_collector import RelationshipCollector
+from app.handlers.blocks.summary import Summary
+from app.handlers.blocks.section_summary import SectionSummary
+from app.handlers.blocks.calculated_summary import CalculatedSummary
+from app.questionnaire.location import InvalidLocationException
+
+BLOCK_MAPPINGS = {
+    'Question': Question,
+    'ConfirmationQuestion': Question,
+    'ListCollector': ListCollector,
+    'ListAddQuestion': ListAddQuestion,
+    'ListEditQuestion': ListEditQuestion,
+    'ListRemoveQuestion': ListRemoveQuestion,
+    'PrimaryPersonListCollector': PrimaryPersonListCollector,
+    'PrimaryPersonListAddOrEditQuestion': PrimaryPersonQuestion,
+    'RelationshipCollector': RelationshipCollector,
+    'Introduction': Content,
+    'Interstitial': Content,
+    'Confirmation': Content,
+    'Summary': Summary,
+    'SectionSummary': SectionSummary,
+    'CalculatedSummary': CalculatedSummary,
+}
+
+
+class BlockFactory:
```

Comment:
I still don't see the benefit of this being a class - it would be simpler to have it as a module level function

Reply:
Yeah, I'll take a look at this.

Reply:
I've renamed `get_handler` to `get_block_handler` and moved the functionality into `block.py`.

Reply:
This is now just a method.